### PR TITLE
Renamed wrap layer to perspective layer

### PR DESCRIPTION
**Pull request by ARYA-BHARDWAJ**
This PR is regarding  issue no #1191.
 - wrap.cpp , wrap.h is renamed to perspective.cpp and perspective.h in folder /synfig-core/src/modules/lyr_std
- other required changes are made

### CODE REVIEW

File: synfig-core/src/modules/lyr_std/main.cpp [hunk 1]
```diff
@@ -83,34 +83,35 @@ using namespace lyr_std;
...
-	MODULE_COPYRIGHT(SYNFIG_COPYRIGHT)
+MODULE_NAME("Standard Layers")
+MODULE_DESCRIPTION("Provides a basic set of standard layers")
+MODULE_AUTHOR("Robert B. Quattlebaum")
+MODULE_VERSION("1.0")
+MODULE_COPYRIGHT(SYNFIG_COPYRIGHT)
 MODULE_DESC_END
 
 MODULE_INVENTORY_BEGIN(liblyr_std)
-	BEGIN_LAYERS
```
Comment (ankit-kumar-dwivedi) [inline synfig-core/src/modules/lyr_std/main.cpp]: please reformat this code

File: synfig-core/src/modules/lyr_std/main.cpp [hunk 2]
```diff
@@ -83,34 +83,35 @@ using namespace lyr_std;
 /* === E N T R Y P O I N T ================================================= */
 
 MODULE_DESC_BEGIN(liblyr_std)
-	MODULE_NAME("Standard Layers")
```
Comment (ankit-kumar-dwivedi) [inline synfig-core/src/modules/lyr_std/main.cpp]: indent this as well

**Comment by ARYA-BHARDWAJ** [comment]
Yeah sure i updated it

**Comment by ARYA-BHARDWAJ** [comment]
@ice0 please review

File: synfig-core/src/modules/lyr_std/main.cpp (hunk 2 above)
Comment (ice0) [reply synfig-core/src/modules/lyr_std/main.cpp]: @ARYA-BHARDWAJ please indent here too.

**Pull request closed without merging**